### DISCUSSION TITLE
[Q] Properties of Pivot table is lost on saving the file

### DISCUSSION THREAD

**Discussion by persy97** [exceljs/exceljs#2407] [Q&A]
## 💬 Questions and Help

I have an excel file with some data in one of the sheets and another sheet with pivot table on that data.  On trying to add few rows in the sheets and saving the workbook, sheet with pivot table is losing its properties and that table converts into a normal table. Not sure if this is a bug or I am doing something wrong here!

Please find the code being used below:
```
const ExcelJS = require('exceljs');
var workbook = new ExcelJS.Workbook();
workbook.xlsx.readFile('ABC.xlsx')
    .then(function() {
        var worksheet = workbook.getWorksheet(3);
        var row = worksheet.getRow(8);
        row.getCell(1).value = "Exact";      
        row.getCell(2).value = "ABC";
        row.commit();
        return workbook.xlsx.writeFile('ABC.xlsx');
    });
```

**Comment by fabrykowski** [comment]
See #261
